### PR TITLE
Added Cypress for Integration tests

### PR DESCRIPTION
**Pull request by plug-n-play**
* [ ]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [ ]  Ran `npm test` & fixed newly introduced lint errors.
* [ ]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: cypress/integration/home_spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+describe('Home page Search tests', () => {
+  it('should search shabad Jo Mange Thakur Apne Te with default settings', () => {
+    cy.visit('/')
+
+    cy.get('#search')
+      .clear()
+      .type('bhb')
+
+    // get api response
+    cy.wait(3000)
```
Comment (gurjit03) [inline cypress/integration/home_spec.js:10]: @plug-n-play This is considered kinda antipattern in cypress maybe we can do something else here?

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (gurjit03) [inline cypress/fixtures/example.json:1]: @plug-n-play can prolly delete this fixtures example.json !

**Comment by gurjit03** [comment]
@plug-n-play looks good to me veerji thanks !

File: cypress/integration/home_spec.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+      cy.get('.search-highlight-word')
+        .eq(1)
+        .should('contain', 'hI')
+
+      cy.get('.search-highlight-word')
+        .eq(2)
+        .should('contain', 'bYkuMT')
+    })
+
+    it('should open all search results page on clicking the "Show All Results" button', () => {
```
Comment (tsingh777) [inline cypress/integration/home_spec.js:31]: is this test relying on the previous test?

File: cypress/integration/home_spec.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
+describe('Home page Search tests', () => {
+  describe('First Letter Search from Start(G)', () => {
+    it('should return search results for "bhb" in drop-down', () => {
+      cy.visit('/')
```
Comment (tsingh777) [inline cypress/integration/home_spec.js:4]: nit: prefer semi-colons.

File: package.json [hunk 5]
```diff
@@ -8,6 +8,7 @@
     "server": "node -r esm server",
     "start": "npm run server",
     "dev": "npm run watch",
+    "cypress:open": "cypress open",
```
Comment (tsingh777) [inline package.json:11]: Need to follow up with a script to run this headless in the CI/CD pipeline.

**Comment by saintsoldierx** [comment]
@plug-n-play please resolve the suggestions by Taranjit - then we can merge

File: cypress/integration/home_spec.js (hunk 3 above)
Comment (tsingh777) [reply cypress/integration/home_spec.js:31]: Can follow up on this.

**Comment by plug-n-play** [comment]
@saintsoldierx - this can be now be merged

**Pull request merged**